README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+# Metrics webapp
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+А mobile web application to check a list of metrics (numeric values).
 
-## Available Scripts
+## Screenshot of the page
 
-In the project directory, you can run:
+![screenshot](./src/app.JPG)
 
-### `npm start`
+## Project Decription
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+A mobile web with a list of metrics that can be filtered by the country.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Live Demo
 
-### `npm test`
+[Live Demo Link](https://laughing-newton-9afe3e.netlify.app/#/)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Demo Video
 
-### `npm run build`
+[Live Demo Link](https://www.loom.com/share/f9dc1626a06d4d2da555e00bd3f5bff9)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Build With
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- `React & Redux`
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Getting Started
 
-### `npm run eject`
+To get a local copy up and running follow these simple example steps.
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Instructions
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+$ cd <folder>
```

Comment:
- [ ] [Optional] It would be great if this is in markdown using ` ` .

Reply:
Thank you!

I will fix it.